INSTALL.md

```diff
@@ -32,6 +34,36 @@ How to install
 
 See also [repology's cross-distro index](https://repology.org/metapackage/zsh-syntax-highlighting/versions)
 
+Second, enable zsh-syntax-highlighting by sourcing the script. Running this command on the terminal will add the source line to the end of your .zshrc:
+
+* On most Linux distributions (except perhaps NixOS):
+
+```zsh
+ echo "source /usr/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh"  ${ZDOTDIR:-$HOME}/.zshrc
+```
+
+* NetBSD and OpenBSD:
+
+```zsh
+ echo "source /usr/local/share/zsh-syntax-highlighting/zsh-syntax-highlighting.zsh"  ${ZDOTDIR:-$HOME}/.zshrc
+```
+
+* Mac OS X / Homebrew:
+
+```
+script location for source is unknown. If you do, please submit a documentation patch/issue.
```

Comment:
I recommend to avoid the use of a preformatted block in this case.  This isn't a command to be run, but prose.  Setting it as prose should make the document easier for readers to follow.

Wording nits: capitalize the first letter; don't use "source" as a common noun (if you mean to refer to the `source` builtin, it should be quoted to make it clear it's a proper noun; another possibility is to abstract the builtin's name and say "to be sourced"); it's not clear what the "do" means.